src/Confix.Tool/src/Confix.Tool/Variables/VariablePath.cs

```diff
@@ -1,3 +1,17 @@
 namespace ConfiX.Variables;
 
-public record struct VariablePath(string ProviderName, string Path);
+public record struct VariablePath(string ProviderName, string Path)
```

Comment:
When you make the struct readonly then it does not get copied in the ToString method :)